docs/helm.md

```diff
@@ -0,0 +1,120 @@
+# Deploying to Kubernetes with Helm
+
+[Helm](https://helm.sh/) is a popular Kubernetes package manager. To deploy to Kubernetes with Helm, you'll first need to [initialize it in your cluster](https://docs.helm.sh/using_helm#install-helm).
+
+## Initial Project Structure
+
+All rok8s-scripts configuration lives in a `deploy` directory at the root of your project by default. In this example, we have a simple Python app with a Dockerfile in place.
+
+```plaintext
+app-dir/
+├── deploy/
+│   ├── build.config                         // rok8s-scripts build configs
+│   ├── charts/                              // All HELM chart configuration for the app
```

Comment:
My standard setup has usually been to leave charts in the root of the repo and reference them there.  Is this our normal recommendation to leave them in the deploy directory?

Reply:
It doesn't actually matter all that much, I was just curious.

Reply:
That's interesting, I've always had them in a charts directory, but I guess there's really no need for that